doc/rtd/reference/yaml_examples/ansible_controller.rst

```diff
@@ -0,0 +1,168 @@
+.. _cce-ansible-controller:
```

Comment:
The other pages are all a set of independent examples separated by section headers, but this is a single example separated by section headers. Is there a way we can lay this page out differently so it's obvious this is one big example rather than a set of unrelated examples?

Reply:
This was my attempt to avoid having comments inside the example, but I think in this case it's unavoidable. Changed it back to how it was.

Reply:
Let me know if this is better